xbmc/guilib/guiinfo/VideoGUIInfo.cpp

```diff
@@ -437,6 +437,12 @@ bool CVideoGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
       case LISTITEM_VIDEO_ASPECT:
         value = CStreamDetails::VideoAspectToAspectDescription(tag->m_streamDetails.GetVideoAspect());
         return true;
+      case LISTITEM_VIDEO_WIDTH:
+        value = std::to_string(tag->m_streamDetails.GetVideoWidth());
```

Comment:
I think it is better to use std::to_chars since unlike std::to_string is noexcept?
Ideally this should be a new method in stringutils that returns std::optional<std::string> depending on the fact it successfully converts the value to the string or not. Then you could return true or false depending on this fact instead of blindingly return true regardless of the conversion success

Reply:
I can't think of a circumstance where to_string would throw an exception.  GetVideoWidth always returns an integer, even if zero.  The OSMC skin traps zeros and falls back to displaying the generic mode strings. To do that here would be rather complicated for the case where only one of width or height returned zero.

Reply:
Thanks for looking into it. If the value is always guaranteed to be an integer you can disregard the comment.